runtime/j9vm/java11vmi.c

```diff
@@ -771,7 +839,7 @@ JVM_DefineModule(JNIEnv * env, jobject module, jboolean isOpen, jstring version,
 				J9Module *j9mod = createModule(currentThread, modObj, classLoader, moduleName);
 				if (NULL != j9mod) {
 					BOOLEAN success = FALSE;
-					UDATA rc = addModuleDefinition(currentThread, j9mod, packages, (U_32) numPackages, version);
+					UDATA rc = addModuleDefinition(currentThread, j9mod, (const char* const*)packages, (U_32) numPackages, version);
```

Comment:
The cast here should not be needed; did some compiler tell you otherwise?

Reply:
Maybe, I've been through so many iterations. `packages` is declared as `const char* *` not `const char* const*`, but I can remove it and see.

Reply:
In general, there's an implicit cast from `T *` to `T const *` (`T` is `const char *` here).